packages/app/src/kinesis.ts

```diff
@@ -1,18 +1,19 @@
-import type { Kinesis } from 'aws-sdk';
 import type {
-	PutRecordsOutput,
+	Kinesis,
+	PutRecordsCommandOutput,
 	PutRecordsRequestEntry,
-} from 'aws-sdk/clients/kinesis';
+} from '@aws-sdk/client-kinesis';
+import { PutRecordsCommand } from '@aws-sdk/client-kinesis';
 import type { PublishableStructuredLogData } from './model';
 
 export async function putKinesisRecords(
 	kinesis: Kinesis,
 	streamName: string,
 	logEntries: PublishableStructuredLogData[],
-): Promise<PutRecordsOutput[]> {
+): Promise<PutRecordsCommandOutput[]> {
 	const records: PutRecordsRequestEntry[] = logEntries.map((entry) => {
 		return {
-			Data: JSON.stringify(entry),
+			Data: new TextEncoder().encode(JSON.stringify(entry)),
```

Comment:
This is one of the only places that wasn't a one-to-one translation from v2 to v3 of the SDK. I've verified that log entries are arriving properly formatted in ELK, so assume we're properly putting records onto the kinesis stream.